src/components/alarm/AlarmController.cpp

```diff
@@ -0,0 +1,109 @@
+/*  Copyright (C) 2021 JF, Adam Pigg, Avamander
+    This file is part of InfiniTime.
+    InfiniTime is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    InfiniTime is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+#include "AlarmController.h"
+#include "systemtask/SystemTask.h"
+#include "app_timer.h"
+#include "task.h"
+#include <chrono>
+
+using namespace Pinetime::Controllers;
+using namespace std::chrono_literals;
+
+AlarmController::AlarmController(Controllers::DateTime& dateTimeController) : dateTimeController {dateTimeController} {
+}
+
+APP_TIMER_DEF(alarmAppTimer);
+
+namespace {
+  void SetOffAlarm(void* p_context) {
+    auto* controller = static_cast<Pinetime::Controllers::AlarmController*>(p_context);
+    if (controller != nullptr)
```

Comment:
Missing braces are not consistent with the rest of the code

Reply:
Added braces